Part2_Mapplots/ex2_2_psyplot.ipynb

```diff
@@ -193,9 +193,10 @@
     "ax = plt.axes(projection=ccrs.Robinson())\n",
```

Comment:
<p>Introduce some basic methods like .update, .maplot etc.. e.g. How the syntax works, before they are used in the code. Otherwise it would confuse people</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>